cdap-docs/admin-manual/source/installation/cdap-site.rst

```diff
@@ -130,40 +130,43 @@ see the :ref:`configuration-security` section.
    * - ``explore.inactive.operation.timeout.secs``
      - ``3600``
      - Timeout value in seconds for a SQL operation which has no more results to be fetched
-   * - ``gateway.boss.threads``
+   * - ``router.boss.threads``
      - ``1``
      - Number of Netty server boss threads
-   * - ``gateway.connection.backlog``
+   * - ``router.connection.idle.timeout.secs``
+     - ``15``
+     - The number of seconds after an http request completes that idle router connections are closed
```

Comment:
updated the rst file.
Note that its lowercased in on develop and release/3.3 branches, though. Probably not a big deal.